### DISCUSSION TITLE
Can't style ErrorMessage Component for some reason.

### DISCUSSION THREAD

**Discussion by ladygo93** [react-hook-form/react-hook-form#7134] [Q&A]
Hi, 
For some reason can't style ErrorMessage Component as shown on image. For some reason message is not even in the tags I put it in.

![Screenshot (40)](https://user-images.githubusercontent.com/65852371/143233374-a3f778b1-1aa7-4926-aea2-15af8a53e6d5.png)

**Comment by creatorGod003** [comment]
Same issue I am also facing right now.
